### PR TITLE
Div text change when we change modes with the button

### PR DESCRIPTION
**Pull request by Kledis98**

### CODE REVIEW

File: src/app/header/header.component.html [hunk 1]
```diff
@@ -189,8 +191,16 @@ <h5 id="offcanvasStartLabel">penda</h5>
                         />
                         <span class="dark-mode-slider"></span>
                     </label>
-                </div>
+                </div>        
             </div>
+            <ng-template 
+            [ngIf]="darkModeService.isDarkMode" 
+            [ngIfElse]="textToChange">
```
Comment (AndiBraimllari) [inline src/app/header/header.component.html:198]: `textToChange` could be used anywhere else! The name should be more meaningful, e.g. something like `otherModeText`.

File: src/app/header/header.component.html [hunk 2]
```diff
@@ -189,8 +191,16 @@ <h5 id="offcanvasStartLabel">penda</h5>
...
+                <span class="dark-mode-theme">Përdor pamjen e çelur</span>
+            </ng-template>
+            <ng-template #textToChange>
+                <span class="light-mode-theme">Përdor pamjen e errët</span>
```
Comment (AndiBraimllari) [inline src/app/header/header.component.html:202]: Capitalize the words, i.e. `Përdor Pamjen e Errët` instead, same for light mode.

**Comment by AndiBraimllari** [comment]
Would suggest you run prettier here, I see two empty lines of code (and potentially other formatting changes required).

File: src/app/header/header.component.html (hunk 2 above)
Comment (Kledis98) [reply src/app/header/header.component.html:202]: Sure

**Pull request merged**